.github/workflows/issue.yml

```
@@ -1,13 +1,66 @@
+# NOTE!
+#
+# Please read the README.md file in this directory that defines what should 
+# be placed in this file
+#
+##############################################################################
+##############################################################################
+
+name: Issue Workflow
 on:
   issues:
       types: ['opened']
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 6

(indentation)